core/gui/src/environments/environment.default.ts

```diff
@@ -100,6 +100,11 @@ export const defaultEnvironment = {
    * default data transfer batch size for workflows
    */
   defaultDataTransferBatchSize: 400,
+
+  /**
+   * Whether email notification is enabled as default
+   */
+  defaultEmailNotificationEnabled: false,
```

Comment:
Use `emailNotificationEnabled` instead of `defaultEmailNotificationEnabled`? `Default` is confusing here.

Reply:
This brings up another issue: we want to avoid situations where a flag is set but can be overridden, rendering it ineffective. I believe that if the user disables Email Notifications, they should be disabled globally. Currently, this flag only disables notifications for workflows, while notifications for sharing are still active. To address this, either rename the flag to indicate its limited scope or modify its behavior to apply globally.

Reply:
Thanks. I'll rename it to `workflowEmailNotificationEnabled`